streaming_form_data/targets.py

```diff
@@ -35,20 +51,19 @@ def value(self):
 
 
 class FileTarget(BaseTarget):
-    def __init__(self, filename):
+    def __init__(self, filename, allow_overwrite=True):
```

Comment:
The `filename` attribute here is now in conflict with the `filename` defined in `BaseTarget.__init__`.

Renaming `filename` here to `outfile` should avoid this conflict.

Reply:
I'm afraid the same error may be made by end users writing custom targets classes (or who already wrote).

I guess we may break someone's code who derived from FileTarget.

Maybe we'd better somehow prevent such errors by renaming base class member?

It could have name 'content_disposition_filename' or 'multipart_filename' or...

Reply:
Good point. `multipart_filename` should work.